crates/compiler/test_gen/src/helpers/llvm.rs

```diff
@@ -335,6 +335,8 @@ pub fn helper<'a>(
     let (main_fn_name, delayed_errors, module) =
         create_llvm_module(arena, src, config, context, target, function_kind);
 
+    //module.print_to_file(std::path::Path::new("/home/username/gitrepos/roc4/roc/str_to_u128.ll")).unwrap();
```

Comment:
should this be removed?

Reply:
I'd like to leave it there for easy debugging, I've added a clarifying comment.